src/dlgTriggerEditor.cpp

```diff
@@ -1111,21 +1111,10 @@ void dlgTriggerEditor::slot_search_triggers( const QString s )
             {
                 QStringList sl;
                 sl << "Key" << key->getName() << "name";
-                if( ! parent )
```

Comment:
Well it's `dlgTriggerEditor::slot_search_triggers` so this is the code that shows the search results when you search in the trigger editor. The existing search behaviour worked as-is from my scratch test so I'm happy.

Ideally we'd have the GUI automation tests for this but I'm only one person. If anyone is interested in helping get them fleshed out more, let me know.